docs/develop/common-coding-issues.md

```diff
@@ -35,6 +41,17 @@ You can identify a property that must have its subproperties set with a JSON str
 - Read-only property: Subproperties can be set through navigation.
 - Writable property: Subproperties must be set with a JSON struct (and cannot be set through navigation).
 
+## Excel Range limits
```

Comment:
I'm wondering if (for discoverability), it'd be better to add this information to the [Ranges (advanced)](https://docs.microsoft.com/en-us/office/dev/add-ins/excel/excel-add-ins-ranges-advanced) article, instead of here in this coding issues article?  Or, perhaps define the contents of the section in an include file, and include it both here in the coding issues article and also in the Excel Ranges (advanced) article? Thoughts?

Reply:
There's a section in the Excel Core Concepts doc that mentions the 5MB limit. I'll replace that with a link to this section. I'll also add a link to the new coding issues page from the **See Also** list.